yt_dlp/extractor/rule34video.py

```diff
@@ -60,6 +116,13 @@ def _real_extract(self, url):
             'thumbnail': thumbnail,
             'duration': parse_duration(duration),
             'age_limit': 18,
+            'like_count': like_count,
+            'comment_count': comment_count,
+            'description': description,
+            'creator': artists if len(artists) > 0 else None,
```

Comment:
`creator` should be a string
```suggestion
            'creator': join_nonempty(*artists, delim=', ')
```
Import `join_nonempty` from `..utils`.

Reply:
Done.